eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -826,7 +826,7 @@ case class Commitments(channelId: ByteVector32,
   /**
    * When reconnecting, we drop all unsigned changes.
    */
-  def discardUnsignedUpdates(implicit log: LoggingAdapter): Commitments = {
+  def discardUnsignedUpdates()(implicit log: LoggingAdapter): Commitments = {
```

Comment:
Actually this method can be removed entirely, we only need it at the `MetaCommitments` level